packages/truffle/webpack.config.js

```diff
@@ -150,180 +158,49 @@ module.exports = {
         to: "initSource"
       },
       {
-        from: path.join(
-          __dirname,
-          "../..",
-          "node_modules",
-          "@truffle/core",
-          "lib",
-          "testing",
-          "Assert.sol"
-        )
+        from: path.join(truffleLibraryDirectory, "Assert.sol")
       },
       {
-        from: path.join(
-          __dirname,
-          "../..",
-          "node_modules",
-          "@truffle/core",
-          "lib",
-          "testing",
-          "AssertAddress.sol"
-        )
+        from: path.join(truffleLibraryDirectory, "AssertAddress.sol")
       },
       {
-        from: path.join(
-          __dirname,
-          "../..",
-          "node_modules",
-          "@truffle/core",
-          "lib",
-          "testing",
-          "AssertAddressArray.sol"
-        )
+        from: path.join(truffleLibraryDirectory, "AssertAddressArray.sol")
       },
       {
-        from: path.join(
-          __dirname,
-          "../..",
-          "node_modules",
-          "@truffle/core",
-          "lib",
-          "testing",
-          "AssertAddressPayableArray.sol"
```

Comment:
Why'd this get removed? This sounds like something accidental. Do we care about this type?

Reply:
This was not an accident, I don't think it is used. It was commented out in the list [here](https://github.com/trufflesuite/truffle/blob/develop/packages/core/lib/testing/TestSource.js#L78). Should we add it back in?

Reply:
We can do this if someone complains